### DISCUSSION TITLE
Feature Suggestion: QToggle button 'flat' style needed

### DISCUSSION THREAD

**Discussion by tohagan** [quasarframework/quasar#12209] [Ideas / Proposals] [answered]
I've recently tried to use the QToggle button on a toolbar but it looked rather clunky beside the other flat buttons. 
I think it really needs a 'flat' style option like the one I see on https://staging.vuejs.org/ for dark/light mode toggling. 

![image](https://user-images.githubusercontent.com/301525/151284023-da67a528-ca36-466d-9f46-533c592c0749.png)

![image](https://user-images.githubusercontent.com/301525/151283946-dd63b1bf-37b2-4408-8749-b3cd0276ff46.png)

I realise that the current design is intended to be Material Design compliant.
- https://material.io/components/switches

**Comment by metalsadman** [comment, marked as answer]
can customize css somewhat like this https://codepen.io/metalsadman/pen/LYxxMmo
